static/.gitignore

```diff
@@ -1,2 +1,10 @@
 min/
 source-map/
+/js/bundle.js
```

Comment:
I think `/js/bundle.js` no longer exists, and was replaced by `/webpack-bundles`.